Demo/app/src/main/java/com/juanocampo/swipeable/swipeablelist/swpeable/fragment/SwipeableFragment.java

```diff
@@ -0,0 +1,38 @@
+package com.juanocampo.swipeable.swipeablelist.swpeable.fragment;
+
+import android.support.annotation.NonNull;
+import android.support.v4.app.Fragment;
+
+import com.juanocampo.swipeable.swipeablelist.fragment.swipeable_implementation.RecyclerListAdapter;
+import com.juanocampo.swipeable.swipeablelist.swpeable.adapter.SwipeableHelperAdapter;
+
+/**
+ * Created by juan.ocampo on 09/08/2016.
+ */
+
+public abstract class SwipeableFragment extends Fragment implements RecyclerListAdapter.SwipeAdapterActions{
```

Comment:
i would remove this class all together.  If child fragment needs to imeplement getAdapter, then change this to a interface.